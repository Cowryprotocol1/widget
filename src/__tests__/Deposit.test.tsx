import { fireEvent, render } from "@testing-library/react";
import DepositModal from "../deposit/index";
import React from 'react';
const minDeposit: string = "999";

describe("Withdrawal Component", () => {
  // defining the component
  const testComponent =()=> render(<DepositModal minDeposit={minDeposit}/>)

  it(" ❌ renders DepositModal Component correctly'", () => {
   const {asFragment} = testComponent();
   expect(asFragment()).toMatchSnapshot();
  });

  it(" ❌ count number of input in the form'", async () => {
    let component = testComponent();
    const formInputs = component.getAllByTestId('form')
    expect(await formInputs).toHaveLength(4);
   });

  it(" ❌ Submit Form Data'", async() => {
    const {getByTestId, getByText, getByPlaceholderText} = testComponent();

    const amount: any = getByPlaceholderText("Deposit Amount");
    fireEvent.change(amount, { target: { value: "1000" } });
    expect(amount.value).toBe("1000");

    const bank: any = getByPlaceholderText("Deposit Bank Name");
    fireEvent.change(bank, { target: { value: "GTB" } });
    expect(bank.value).toBe("GTB");

    const desc: any = getByPlaceholderText("Wallet Address");
    fireEvent.change(desc, { target: { value: "GRXISJSJ2DJ4H2N44NK2N2" } });
    expect(desc.value).toBe("GRXISJSJ2DJ4H2N44NK2N2");

    const accNum: any = getByPlaceholderText("Transaction Description");
    fireEvent.change(accNum, { target: { value: "TEST DESCRIPTION" } });
    expect(accNum.value).toBe("TEST DESCRIPTION");

    const button = getByTestId('submit_btn')
    fireEvent.click(button);
    expect(await getByText('Confirm Deposit')).toBeInTheDocument();

  });
});