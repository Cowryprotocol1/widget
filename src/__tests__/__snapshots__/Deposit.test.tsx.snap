// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Withdrawal Component  ‚ùå renders DepositModal Component correctly' 1`] = `
<DocumentFragment>
  <div
    class="bg-white w-[40%] md:w-[35%] py-4 px-[2.5%] rounded"
  >
    <div
      class="w-full p-3 flex flex-row bg-[#062638] rounded-lg"
    >
      <div
        class="flex text-sm flex-col justify-start"
      >
        <p
          class="text-white font-thin mb-2 text-left"
        >
          Make Deposit
        </p>
        <p
          class="text-white font-thin text-left md:text-xs text-[0.7rem]"
        >
          Please kindly fill in the necessary details for your deposit below.
        </p>
      </div>
      <div>
        <svg
          fill="none"
          height="100"
          viewBox="0 0 114 100"
          width="114"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            clip-path="url(#clip0_578_3518)"
          >
            <path
              d="M15.7643 38.9147C23.8331 38.9147 30.3743 32.3926 30.3743 24.347C30.3743 16.3015 23.8331 9.77936 15.7643 9.77936C7.69541 9.77936 1.1543 16.3015 1.1543 24.347C1.1543 32.3926 7.69541 38.9147 15.7643 38.9147Z"
              fill="#F4F4F4"
            />
            <g
              opacity="0.05"
            >
              <path
                d="M15.7637 35.8956C22.1604 35.8956 27.3458 30.7251 27.3458 24.347C27.3458 17.9689 22.1604 12.7985 15.7637 12.7985C9.36712 12.7985 4.18164 17.9689 4.18164 24.347C4.18164 30.7251 9.36712 35.8956 15.7637 35.8956Z"
                fill="#292A2E"
              />
            </g>
            <path
              d="M11.9727 28.4721L13.0531 26.8015C13.9092 27.4558 14.6585 27.7899 15.455 27.7899C16.4099 27.7899 16.83 27.395 16.83 26.6305C16.83 24.9023 12.383 24.9335 12.383 21.9022C12.383 19.9536 13.7367 18.7124 15.8714 18.7124C17.3125 18.7124 18.2916 19.3054 19.0784 20.1375L17.8393 21.5528C17.227 20.9975 16.7078 20.7051 16.0043 20.7051C15.1932 20.7051 14.7542 21.0309 14.7542 21.7856C14.7542 23.388 19.2011 23.2476 19.2011 26.458C19.2011 28.387 17.921 29.7826 15.455 29.7826C14.3911 29.7822 12.9241 29.3317 11.9727 28.4721ZM14.9358 17.0658H16.5792V19.485H14.9358V17.0658ZM14.9358 28.774H16.5792V31.4283H14.9358V28.774Z"
              fill="#F5FFF9"
            />
            <path
              d="M132.095 7.4975H30.7616C28.2732 7.4975 26.2559 9.50896 26.2559 11.9902V85.9668C26.2559 88.4481 28.2732 90.4595 30.7616 90.4595H132.095C134.583 90.4595 136.6 88.4481 136.6 85.9668V11.9902C136.6 9.50896 134.583 7.4975 132.095 7.4975Z"
              fill="#F5FFF9"
            />
            <path
              d="M132.094 90.6091H30.7614C29.527 90.6077 28.3436 90.1181 27.4708 89.2478C26.5979 88.3775 26.1069 87.1975 26.1055 85.9667V11.9902C26.1069 10.7594 26.5978 9.57934 27.4707 8.709C28.3436 7.83867 29.527 7.3491 30.7614 7.34772H132.094C133.329 7.34907 134.512 7.83862 135.385 8.70896C136.258 9.5793 136.749 10.7594 136.75 11.9902V85.9667C136.749 87.1975 136.258 88.3775 135.385 89.2478C134.512 90.1182 133.329 90.6078 132.094 90.6091ZM30.7614 7.64723C29.6067 7.6485 28.4995 8.10647 27.683 8.92066C26.8664 9.73485 26.4071 10.8388 26.4059 11.9902V85.9667C26.4071 87.1181 26.8664 88.222 27.683 89.0362C28.4995 89.8504 29.6067 90.3084 30.7614 90.3096H132.094C133.249 90.3084 134.356 89.8504 135.173 89.0362C135.989 88.222 136.449 87.1181 136.45 85.9667V11.9902C136.449 10.8388 135.989 9.73485 135.173 8.92066C134.356 8.10647 133.249 7.6485 132.094 7.64723H30.7614Z"
              fill="#292A2E"
            />
            <path
              d="M45.952 48.0454C45.8929 48.0455 45.8342 48.0339 45.7796 48.0113C45.7249 47.9887 45.6753 47.9556 45.6335 47.9138L43.5308 45.8172C43.4487 45.7324 43.4033 45.619 43.4043 45.5012C43.4053 45.3834 43.4527 45.2707 43.5363 45.1874C43.6198 45.1041 43.7328 45.0568 43.851 45.0558C43.9691 45.0548 44.0829 45.1001 44.1679 45.1819L46.2706 47.2785C46.3336 47.3413 46.3765 47.4214 46.3939 47.5085C46.4112 47.5957 46.4023 47.686 46.3682 47.7681C46.3341 47.8501 46.2764 47.9203 46.2023 47.9697C46.1282 48.019 46.0411 48.0454 45.952 48.0454Z"
              fill="#0D2A3B"
            />
            <path
              d="M43.8489 48.0454C43.7598 48.0454 43.6727 48.019 43.5986 47.9697C43.5246 47.9203 43.4668 47.8501 43.4327 47.7681C43.3986 47.686 43.3897 47.5957 43.4071 47.5085C43.4245 47.4214 43.4674 47.3413 43.5304 47.2785L45.6331 45.1819C45.718 45.1001 45.8319 45.0548 45.95 45.0558C46.0681 45.0568 46.1811 45.1041 46.2647 45.1874C46.3482 45.2707 46.3956 45.3834 46.3966 45.5012C46.3977 45.619 46.3522 45.7324 46.2702 45.8172L44.1675 47.9138C44.1257 47.9556 44.0761 47.9887 44.0214 48.0113C43.9667 48.0339 43.9081 48.0455 43.8489 48.0454Z"
              fill="#0D2A3B"
            />
            <path
              d="M106.157 68.0846L105.557 69.4574L109.25 71.063L109.851 69.6903L106.157 68.0846Z"
              fill="#0D2A3B"
            />
            <path
              d="M107.322 31.2801L103.848 33.3157L104.609 34.6068L108.083 32.5713L107.322 31.2801Z"
              fill="#0D2A3B"
            />
            <path
              d="M49.4301 34.766L48.9648 36.1899L52.7963 37.4347L53.2615 36.0108L49.4301 34.766Z"
              fill="#0D2A3B"
            />
            <path
              d="M53.9154 58.6481L50.6152 60.9503L51.4758 62.1768L54.776 59.8746L53.9154 58.6481Z"
              fill="#0D2A3B"
            />
            <path
              d="M81.4288 75.042C94.6616 75.042 105.389 64.3457 105.389 51.1513C105.389 37.9568 94.6616 27.2606 81.4288 27.2606C68.196 27.2606 57.4688 37.9568 57.4688 51.1513C57.4688 64.3457 68.196 75.042 81.4288 75.042Z"
              fill="#0D2A3B"
              opacity="0.4"
            />
            <path
              d="M81.4284 67.8593C90.6828 67.8593 98.1849 60.3788 98.1849 51.1513C98.1849 41.9237 90.6828 34.4432 81.4284 34.4432C72.174 34.4432 64.6719 41.9237 64.6719 51.1513C64.6719 60.3788 72.174 67.8593 81.4284 67.8593Z"
              fill="#0D2A3B"
            />
            <path
              d="M78.0011 58.3191C77.8695 58.3192 77.7393 58.2927 77.6183 58.2413C77.4972 58.1898 77.3879 58.1144 77.297 58.0196L71.9635 52.485C71.873 52.3932 71.8017 52.2844 71.7536 52.1649C71.7056 52.0454 71.6818 51.9175 71.6837 51.7888C71.6855 51.6601 71.713 51.533 71.7644 51.4149C71.8159 51.2968 71.8903 51.1901 71.9834 51.1009C72.0765 51.0117 72.1864 50.9418 72.3068 50.8952C72.4272 50.8487 72.5556 50.8264 72.6847 50.8297C72.8138 50.833 72.9409 50.8617 73.0588 50.9143C73.1766 50.9669 73.2828 51.0423 73.3712 51.1362L78.0512 55.9925L90.2865 45.0484C90.3817 44.9616 90.4932 44.8944 90.6145 44.8509C90.7359 44.8073 90.8647 44.7881 90.9935 44.7945C91.1224 44.8008 91.2486 44.8326 91.3651 44.8879C91.4815 44.9432 91.5859 45.0209 91.672 45.1166C91.7581 45.2124 91.8244 45.3242 91.8669 45.4456C91.9095 45.567 91.9275 45.6956 91.9199 45.824C91.9123 45.9524 91.8793 46.078 91.8227 46.1936C91.7661 46.3092 91.6872 46.4124 91.5903 46.4974L78.6529 58.0698C78.474 58.2304 78.2418 58.3193 78.0011 58.3191Z"
              fill="#F5FFF9"
            />
            <path
              d="M136.6 11.9896V16.5104H26.2559V11.9896C26.2559 10.7981 26.7306 9.65533 27.5756 8.81278C28.4206 7.97023 29.5666 7.49689 30.7616 7.49689H132.094C133.289 7.49689 134.435 7.97023 135.28 8.81278C136.125 9.65533 136.6 10.7981 136.6 11.9896Z"
              fill="#292A2E"
            />
            <path
              d="M5.63133 91.0014C4.72399 91.0012 3.83011 90.7825 3.02578 90.3638C2.22144 89.9451 1.53045 89.3389 1.01162 88.5967C0.492797 87.8545 0.161488 86.9982 0.0458944 86.1009C-0.0696994 85.2036 0.0338437 84.2916 0.347712 83.4428C0.66627 82.5809 8.83734 61.5492 10.2604 57.8941L20.0937 63.5999C19.2939 65.6553 16.3961 73.4457 14.0681 79.7215L17.9357 79.3021L19.153 89.5697L6.24157 90.9693C6.03888 90.991 5.83518 91.0016 5.63133 91.0014Z"
              fill="#D6D6D6"
            />
            <path
              d="M12.3815 91.3263L10.6641 99.8503H44.0217L42.1003 91.1643L12.3815 91.3263Z"
              fill="#262626"
            />
            <path
              d="M37.7951 53.5205C37.7606 53.516 36.8144 57.0667 29.4294 56.5141C24.3169 56.1322 23.0703 52.6623 22.9531 52.6623C23.5043 52.5455 24.3995 52.4242 24.6443 52.2939C25.4493 51.8626 26.0357 51.3376 26.331 50.2108C26.5618 49.3296 26.6794 45.9375 26.6794 44.2355L34.5465 45.9981C34.4368 47.918 33.6694 52.3688 36.3593 53.1565C36.8069 53.2868 37.289 53.4066 37.7951 53.5205Z"
              fill="#E98862"
            />
            <path
              d="M34.2747 49.7726C31.1734 50.0167 27.1894 48.7109 26.4326 44.5851L25.7498 40.8624C24.993 36.7366 26.5529 32.9957 30.4489 32.2852C34.3447 31.5748 37.8703 34.3883 38.627 38.5141L38.9293 40.1623C39.2192 41.7453 39.8078 49.3374 34.2747 49.7726Z"
              fill="#E98862"
            />
            <path
              d="M19.5939 53.2335C16.8069 53.4397 13.9651 53.7406 11.9144 55.6337C9.44464 57.9136 8.99707 60.7169 10.5727 68.4315C11.9284 75.0695 11.3907 77.9702 12.2975 90.2192C12.3243 90.5801 12.3815 91.3263 12.3815 91.3263L42.1947 91.2771C42.1947 91.2771 44.9883 75.0258 46.5306 67.5303C47.7035 61.8318 50.7842 54.9141 37.7948 53.52C37.7606 53.5162 27.3428 52.6602 19.5939 53.2335Z"
              fill="#D6D6D6"
            />
            <path
              d="M12.3806 91.4014C12.3612 91.4014 12.3425 91.3939 12.3285 91.3805C12.3145 91.367 12.3062 91.3487 12.3055 91.3293L11.3711 67.1472C11.3704 67.1274 11.3776 67.1081 11.3912 67.0935C11.4047 67.079 11.4235 67.0704 11.4434 67.0696C11.4533 67.0688 11.4633 67.0701 11.4728 67.0733C11.4822 67.0766 11.4909 67.0817 11.4982 67.0885C11.5055 67.0953 11.5114 67.1035 11.5153 67.1126C11.5193 67.1217 11.5214 67.1316 11.5213 67.1415L12.4557 91.3237C12.4564 91.3435 12.4492 91.3628 12.4357 91.3774C12.4221 91.3919 12.4033 91.4005 12.3834 91.4012L12.3806 91.4014Z"
              fill="#0C0C0C"
            />
            <path
              d="M60.2194 61.8676L62.314 57.1828C62.36 57.0798 62.4271 56.9876 62.5108 56.9119C62.5946 56.8362 62.6932 56.7787 62.8005 56.7431C62.9077 56.7076 63.0212 56.6946 63.1338 56.7052C63.2463 56.7158 63.3554 56.7496 63.4541 56.8045C63.6333 56.9115 63.7663 57.0811 63.8272 57.2803C63.8881 57.4796 63.8726 57.6943 63.7836 57.8827L63.3346 58.8872L66.2011 56.3745C66.5149 56.09 67.0828 55.9402 67.3802 56.3122C67.5039 56.448 67.578 56.6215 67.5901 56.8046C67.6022 56.9877 67.5518 57.1694 67.447 57.3202C67.4505 57.3224 67.4536 57.325 67.4564 57.328C67.4582 57.3306 67.4599 57.3333 67.4613 57.3361L67.9452 56.8969C68.0348 56.8156 68.1416 56.7552 68.2576 56.7202C68.3736 56.6853 68.496 56.6765 68.6159 56.6947C68.7357 56.7129 68.8499 56.7575 68.9503 56.8252C69.0507 56.893 69.1346 56.9823 69.196 57.0865C69.2894 57.2526 69.3227 57.4456 69.2904 57.6332C69.2582 57.8209 69.1623 57.9918 69.0188 58.1174L68.8128 58.3043C68.8154 58.3127 68.8165 58.3215 68.8161 58.3304C68.9847 58.2363 69.1813 58.205 69.3708 58.2422C69.5603 58.2794 69.7304 58.3826 69.8507 58.5333C69.9773 58.7021 70.0355 58.9123 70.0136 59.122C69.9917 59.3317 69.8913 59.5254 69.7325 59.6646L69.2218 60.1288C69.2337 60.1273 69.2457 60.1287 69.257 60.1328C69.2682 60.1368 69.2783 60.1436 69.2864 60.1523C69.2955 60.1632 69.3013 60.1764 69.3031 60.1905C69.4765 60.1483 69.659 60.164 69.8225 60.2353C69.9861 60.3066 70.1217 60.4294 70.2084 60.585C70.2962 60.7503 70.3255 60.9403 70.2914 61.1243C70.2574 61.3082 70.162 61.4753 70.0208 61.5985L65.7051 65.5163C65.1013 66.0644 64.3471 66.4202 63.5392 66.5382C62.7313 66.6561 61.9065 66.5308 61.1704 66.1783C60.5358 65.8743 60.045 65.3356 59.8024 64.6766C59.5598 64.0176 59.5844 63.2902 59.8711 62.6491L60.2194 61.8676Z"
              fill="#E98862"
            />
            <g
              opacity="0.2"
            >
              <path
                d="M64.621 59.8207C64.6343 59.8352 64.6528 59.8439 64.6724 59.8449C64.6921 59.8459 64.7114 59.8391 64.7261 59.8261L67.4493 57.4331C67.4566 57.4266 67.4627 57.4187 67.467 57.4099C67.4714 57.401 67.4739 57.3914 67.4745 57.3816C67.4751 57.3718 67.4738 57.362 67.4706 57.3527C67.4674 57.3434 67.4624 57.3348 67.4559 57.3274C67.4493 57.32 67.4414 57.314 67.4325 57.3097C67.4237 57.3054 67.4141 57.3028 67.4042 57.3022C67.3944 57.3016 67.3845 57.3029 67.3752 57.3061C67.3658 57.3093 67.3572 57.3143 67.3498 57.3208L64.627 59.7136C64.6196 59.7201 64.6135 59.728 64.6092 59.7369C64.6049 59.7457 64.6023 59.7553 64.6017 59.7651C64.6011 59.775 64.6024 59.7848 64.6056 59.7941C64.6088 59.8034 64.6138 59.812 64.6204 59.8194L64.621 59.8207Z"
                fill="#0C0C0C"
              />
            </g>
            <g
              opacity="0.2"
            >
              <path
                d="M65.6467 61.0474C65.66 61.0618 65.6785 61.0705 65.6982 61.0715C65.7178 61.0724 65.7371 61.0657 65.7518 61.0526L68.7908 58.3822C68.7982 58.3757 68.8042 58.3678 68.8086 58.3589C68.8129 58.3501 68.8155 58.3405 68.8161 58.3307C68.8167 58.3208 68.8154 58.311 68.8122 58.3017C68.8089 58.2924 68.8039 58.2838 68.7974 58.2764C68.7839 58.2621 68.7654 58.2534 68.7457 58.2522C68.726 58.251 68.7066 58.2573 68.6914 58.2698L65.6524 60.9403C65.645 60.9468 65.6389 60.9547 65.6346 60.9636C65.6303 60.9724 65.6277 60.982 65.6271 60.9918C65.6265 61.0016 65.6278 61.0115 65.631 61.0208C65.6342 61.0301 65.6392 61.0387 65.6458 61.0461L65.6467 61.0474Z"
                fill="#0C0C0C"
              />
            </g>
            <g
              opacity="0.2"
            >
              <path
                d="M66.7502 62.3822C66.7635 62.3966 66.782 62.4053 66.8016 62.4063C66.8213 62.4073 66.8406 62.4006 66.8553 62.3876L69.2794 60.2574C69.2944 60.2443 69.3035 60.2257 69.3047 60.2059C69.306 60.1861 69.2993 60.1665 69.2861 60.1516C69.2729 60.1367 69.2543 60.1276 69.2344 60.1264C69.2145 60.1251 69.195 60.1318 69.18 60.1449L66.7559 62.2753C66.7485 62.2818 66.7425 62.2897 66.7381 62.2985C66.7338 62.3073 66.7312 62.3169 66.7306 62.3267C66.73 62.3366 66.7313 62.3464 66.7345 62.3557C66.7377 62.365 66.7428 62.3736 66.7493 62.381L66.7502 62.3822Z"
                fill="#0C0C0C"
              />
            </g>
            <path
              d="M37.1484 45.9846C36.5918 43.1955 36.3364 41.3348 36.366 39.3716C37.1502 38.0002 36.944 36.4262 36.7657 35.746C37.5342 36.1641 37.8556 36.2102 38.579 36.2102C38.579 36.2102 37.2822 31.021 32.8746 29.6247C32.0636 29.3102 29.9949 28.9129 27.2472 29.2886C24.9102 29.6082 19.1495 31.3505 18.3258 37.8173V37.821C18.2985 38.0355 17.7455 43.9936 16.6427 46.8523C15.1602 50.6954 13.0791 51.8894 13.1318 56.3233C13.1997 62.0267 23.7153 62.786 30.0585 62.3586C36.4018 61.9311 39.9412 59.8525 39.9875 55.7423C40.0385 51.1371 38.2622 51.5648 37.1484 45.9846Z"
              fill="#0A0A0A"
            />
            <path
              d="M37.1513 43.6468C35.9674 43.8626 34.7759 42.7746 34.5594 41.5942C34.343 40.4138 35.1832 39.5874 36.3672 39.3716L37.1513 43.6468Z"
              fill="#E98862"
            />
            <path
              d="M49.8846 90.1512C48.6937 90.151 47.5335 89.7746 46.5704 89.0761C45.6074 88.3775 44.891 87.3927 44.5242 86.2629L36.6992 62.1318L47.4173 58.6763L51.5325 71.3689L58.6446 60.0914L68.1805 66.0702L54.6513 87.5245C54.1443 88.3284 53.4411 88.991 52.6075 89.4504C51.7739 89.9098 50.8371 90.1509 49.8846 90.1512Z"
              fill="#D6D6D6"
            />
            <path
              d="M43.5713 83.4887C43.5666 83.4887 43.562 83.4883 43.5574 83.4874C43.5379 83.4838 43.5205 83.4726 43.5093 83.4562C43.498 83.4398 43.4937 83.4197 43.4974 83.4001L46.1574 69.0023C46.161 68.9827 46.1723 68.9654 46.1887 68.9542C46.2051 68.9429 46.2253 68.9386 46.2449 68.9422C46.2645 68.9457 46.2819 68.957 46.2932 68.9733C46.3045 68.9897 46.3088 69.0098 46.3052 69.0294L43.6452 83.4274C43.642 83.4446 43.6328 83.4602 43.6193 83.4713C43.6058 83.4825 43.5888 83.4887 43.5713 83.4887Z"
              fill="#0C0C0C"
            />
            <path
              d="M52.4907 74.7675C52.474 74.7675 52.4578 74.7621 52.4447 74.7519C52.4315 74.7418 52.422 74.7276 52.4178 74.7115L51.5317 71.3081C51.5292 71.2986 51.5287 71.2887 51.53 71.2789C51.5314 71.2692 51.5347 71.2598 51.5397 71.2513C51.5447 71.2429 51.5513 71.2354 51.5592 71.2295C51.567 71.2236 51.576 71.2193 51.5855 71.2168C51.5951 71.2143 51.605 71.2138 51.6148 71.2151C51.6246 71.2165 51.634 71.2198 51.6425 71.2248C51.651 71.2297 51.6584 71.2363 51.6643 71.2442C51.6703 71.252 51.6746 71.261 51.6771 71.2705L52.5632 74.6739C52.5682 74.6931 52.5653 74.7135 52.5552 74.7306C52.5451 74.7477 52.5286 74.7601 52.5093 74.7651C52.5032 74.7668 52.497 74.7676 52.4907 74.7675Z"
              fill="#0C0C0C"
            />
            <path
              d="M3.41972 100H153.558C153.598 100 153.636 99.9842 153.665 99.9562C153.693 99.9281 153.709 99.89 153.709 99.8503C153.709 99.8105 153.693 99.7725 153.665 99.7444C153.636 99.7163 153.598 99.7005 153.558 99.7005H3.41972C3.37989 99.7005 3.34169 99.7163 3.31352 99.7444C3.28536 99.7725 3.26953 99.8105 3.26953 99.8503C3.26953 99.89 3.28536 99.9281 3.31352 99.9562C3.34169 99.9842 3.37989 100 3.41972 100Z"
              fill="#D4D4D4"
            />
          </g>
          <defs>
            <clippath
              id="clip0_578_3518"
            >
              <rect
                fill="white"
                height="100"
                rx="12"
                width="114"
              />
            </clippath>
          </defs>
        </svg>
      </div>
    </div>
    <div
      class="flex flex-col items-center gap-6 mt-8 w-[100%]"
    >
      <div
        class="w-[100%] relative mb-2"
      >
        <input
          class="px-4 border-1 h-[45px] bg-[#EDEDED] text-black w-full md:w-[100%] text-xs  font-thin rounded"
          data-testid="form"
          maxlength="100"
          name="address"
          placeholder="Wallet Address"
          type="text"
          value=""
        />
        <p
          class="text-[9px] text-[#414141] absolute top-[-0.5rem] md:top-[-0.9rem] px-1 left-0 md:left-0 bg-transparent"
        >
          Wallet Address
        </p>
      </div>
      <div
        class="w-[100%] relative mb-2"
      >
        <input
          class="px-4 border-1 h-[45px] bg-[#EDEDED] text-black w-full md:w-[100%] text-xs  font-thin rounded"
          data-testid="form"
          maxlength="100"
          name="amount"
          placeholder="Deposit Amount"
          type="text"
          value=""
        />
        <p
          class="text-[9px] text-[#414141] absolute top-[-0.5rem] md:top-[-0.9rem] px-1 left-0 md:left-0 bg-transparent"
        >
          Deposit Amount
        </p>
      </div>
      <div
        class="w-[100%] relative mb-2"
      >
        <input
          class="px-4 border-1 h-[45px] bg-[#EDEDED] text-black w-full md:w-[100%] text-xs  font-thin rounded"
          data-testid="form"
          maxlength="100"
          name="bank"
          placeholder="Deposit Bank Name"
          type="text"
          value=""
        />
        <p
          class="text-[9px] text-[#414141] absolute top-[-0.5rem] md:top-[-0.9rem] px-1 left-0 md:left-0 bg-transparent"
        >
          Deposit Bank Name
        </p>
      </div>
      <div
        class="w-[100%] relative mb-2"
      >
        <input
          class="px-4 border-1 h-[45px] bg-[#EDEDED] text-black w-full md:w-[100%] text-xs  font-thin rounded"
          data-testid="form"
          maxlength="100"
          name="description"
          placeholder="Transaction Description"
          type="text"
          value=""
        />
        <p
          class="text-[9px] text-[#414141] absolute top-[-0.5rem] md:top-[-0.9rem] px-1 left-0 md:left-0 bg-transparent"
        >
          Transaction Description
        </p>
      </div>
      <button
        class="bg-gray-300 text-black rounded px-4 py-2 text-xs"
        data-testid="submit_btn"
        disabled=""
      >
        Proceed
      </button>
    </div>
  </div>
</DocumentFragment>
`;
